integration_tests/tests/sns-s3-tests.ts

```diff
@@ -1,67 +1,69 @@
-import { publishSNS } from "../helpers/snsHelper";
 import { PublishResponse } from "@aws-sdk/client-sns";
-import { getS3ItemsList } from "../helpers/s3Helper";
-import { waitForTrue } from "../helpers/commonHelpers";
 import {
-  snsValidEventPayload,
-  snsInvalidEventNamePayload,
-  snsEventMissingCompIdPayload,
   snsEventInvalidCompId,
-  snsInvalidTimeStampPayload,
+  snsEventMissingCompIdPayload,
+  snsEventMissingEventIdValue,
   snsEventMissingTimestampPayload,
   snsEventWithAdditionalFieldsPayload,
-  snsMissingEventNamePayload,
+  snsInvalidEventNamePayload,
+  snsInvalidTimeStampPayload,
   snsMissingEventIdPayload,
-  snsEventMissingEventIdValue,
-} from "../payloads/snsEventPayload";
-import { resourcePrefix } from "../helpers/envHelper";
+  snsMissingEventNamePayload,
+  snsValidEventPayload,
+} from "../../src/handlers/int-test-support/helpers/payloadHelper";
+import { resourcePrefix } from "../../src/handlers/int-test-support/helpers/envHelper";
+import { listS3Objects } from "../../src/handlers/int-test-support/helpers/s3Helper";
+import { waitForTrue } from "../../src/handlers/int-test-support/helpers/commonHelpers";
+import { publishSNS } from "../../src/handlers/int-test-support/helpers/snsHelper";
 
 let snsResponse: PublishResponse;
 const storageBucket: string = `${resourcePrefix()}-storage`;
 const objectsPrefix = "btm_transactions";
 
 console.log(storageBucket);
 
+const checkForEventId = async (
```

Comment:
Can this method have a more specific name?  Check where?

Reply:
renamed